src/trace/main.go

```diff
@@ -319,8 +321,8 @@ func (ctx *TraceFilterContext) OnHttpStreamDone() {
 	if err != nil {
 		proxywasm.LogError("Failed to get request duration")
 	}
-	reqT := nativeEndian.Uint64(reqTime)
-	reqD := nativeEndian.Uint64(reqDuration)
+	reqT := nativeEndian.Uint64(reqTime) / (Millisecond)
```

Comment:
Can you name the vars so the units will be clear? e.g.:
reqTimeNano, reqDurationNano
reqTMilli, reqDMilli